### ISSUE TITLE
OS X Address Book update fails

### ISSUE THREAD

**Issue by nurtext** [fruux/sabre-katana#289]
Because of Me card could not be updated.

Seems PropertyStorage plugin hasn't been implemented:
http://sabre.io/dav/property-storage/

**Comment by Hywan** [comment]
@nurtext That's correct. I am going to add this soon, this will be fast to implement.

**Comment by nurtext** [comment]
@Hywan Thanks, already tried to implement it by myself but I couldn't get it to work properly.
